### ISSUE TITLE
@bracket.visible ignored for tuplet all-rests

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#409]
When all tuplet children are rests, `@bracket.visible` is ignored.  Notice beat 2 of the first measure does not have a bracket even though `@bracket.visible` for the tuplet is `true` (but it does have a "3"):

<img width="1240" alt="screen shot 2017-01-01 at 21 32 55" src="https://cloud.githubusercontent.com/assets/3487289/21584893/f48a2c46-d069-11e6-8cd4-a360bf099b4a.png">

Test MEI data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title />
   </titleStmt>
   <pubStmt>
    <date>2017-01-01 21:33:03</date>
   </pubStmt>
  </fileDesc>
  <encodingDesc>
   <projectDesc>
    <p>Transcoded from Humdrum with Verovio version 0.9.13-dev-48fd3db</p>
   </projectDesc>
  </encodingDesc>
 </meiHead>
 <music>
  <body>
   <mdiv>
    <score>
     <scoreDef xml:id="scoredef-000000079380399">
      <staffGrp xml:id="m-000000086260025">
       <staffDef xml:id="staffdef-000000006036808" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4" n="1" lines="5" />
      </staffGrp>
     </scoreDef>
     <section xml:id="section-000000154942876">
      <measure xml:id="measure-L4" n="1">
       <staff xml:id="staff-L4F1" n="1">
        <layer xml:id="layer-L4F1" n="1">
         <tuplet xml:id="tuplet-000000133514302" num="3" numbase="2" bracket.visible="true" num.format="count">
          <note xml:id="note-L5F1" dur="8" oct="4" pname="c" accid.ges="n" />
          <note xml:id="note-L6F1" dur="8" oct="4" pname="d" accid.ges="n" />
          <note xml:id="note-L7F1" dur="8" oct="4" pname="e" accid.ges="n" />
         </tuplet>
         <tuplet xml:id="tuplet-000000038004609" num="3" numbase="2" bracket.visible="true" num.format="count">
          <rest xml:id="rest-L8F1" dur="8" />
          <rest xml:id="rest-L9F1" dur="8" />
          <rest xml:id="rest-L10F1" dur="8" />
         </tuplet>
         <tuplet xml:id="tuplet-000000130726293" num="3" numbase="2" bracket.visible="true" num.format="count">
          <note xml:id="note-L11F1" dur="8" oct="4" pname="c" accid.ges="n" />
          <rest xml:id="rest-L12F1" dur="8" />
          <note xml:id="note-L13F1" dur="8" oct="4" pname="d" accid.ges="n" />
         </tuplet>
         <tuplet xml:id="tuplet-000000145832446" num="3" numbase="2" bracket.visible="true" num.format="count">
          <rest xml:id="rest-L14F1" dur="8" />
          <note xml:id="note-L15F1" dur="8" oct="4" pname="e" accid.ges="n" />
          <rest xml:id="rest-L16F1" dur="8" />
         </tuplet>
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L17" right="end">
       <staff xml:id="staff-L17F1" n="1">
        <layer xml:id="layer-L17F1" n="1">
         <tuplet xml:id="tuplet-000000178516366" num="3" numbase="2" bracket.visible="true" num.format="count">
          <rest xml:id="rest-L18F1" dur="8" />
          <note xml:id="note-L19F1" dur="8" oct="4" pname="d" accid.ges="n" />
          <note xml:id="note-L20F1" dur="8" oct="4" pname="e" accid.ges="n" />
         </tuplet>
         <tuplet xml:id="tuplet-000000035207472" num="3" numbase="2" bracket.visible="true" num.format="count">
          <note xml:id="note-L21F1" dur="8" oct="4" pname="c" accid.ges="n" />
          <note xml:id="note-L22F1" dur="8" oct="4" pname="d" accid.ges="n" />
          <rest xml:id="rest-L23F1" dur="8" />
         </tuplet>
         <tuplet xml:id="tuplet-000000130891704" num="3" numbase="2" bracket.visible="true" num.format="count">
          <rest xml:id="rest-L24F1" dur="8" />
          <rest xml:id="rest-L25F1" dur="8" />
          <note xml:id="note-L26F1" dur="8" oct="4" pname="e" accid.ges="n" />
         </tuplet>
         <tuplet xml:id="tuplet-000000177114044" num="3" numbase="2" bracket.visible="true" num.format="count">
          <note xml:id="note-L27F1" dur="8" oct="4" pname="c" accid.ges="n" />
          <rest xml:id="rest-L28F1" dur="8" />
          <rest xml:id="rest-L29F1" dur="8" />
         </tuplet>
        </layer>
       </staff>
      </measure>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

Original Humdrum data:

```
**kern
*M4/4
=1-
12c
12d
12e
12r
12r
12r
12c
12r
12d
12r
12e
12r
=2
12r
12d
12e
12c
12d
12r
12r
12r
12e
12c
12r
12r
==
*-
```

**Comment by craigsapp** [comment]
Here is a sub-case where beams are involved and `@bracket.visible="false"`.  The "3" for beat 2 is not correctly placed on the beam:

<img width="1123" alt="screen shot 2017-01-01 at 21 44 20" src="https://cloud.githubusercontent.com/assets/3487289/21584980/87b37012-d06b-11e6-98aa-4f3d22532fdb.png">

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
 <meiHead>
  <fileDesc>
   <titleStmt>
    <title />
   </titleStmt>
   <pubStmt>
    <date>2017-01-01 21:45:58</date>
   </pubStmt>
  </fileDesc>
  <encodingDesc>
   <projectDesc>
    <p>Transcoded from Humdrum with Verovio version 0.9.13-dev-48fd3db</p>
   </projectDesc>
  </encodingDesc>
  <extMeta>
   <frames xmlns:humxml="http://www.humdrum.org/ns/humxml">
    <metaFrame n="0" token="!!!Xfilter: autobeam" xml:id="loc0">
     <frameInfo>
      <startTime float="0" />
      <frameType>reference</frameType>
      <referenceKey>Xfilter</referenceKey>
      <referenceValue>autobeam</referenceValue>
     </frameInfo>
    </metaFrame>
   </frames>
  </extMeta>
 </meiHead>
 <music>
  <body>
   <mdiv>
    <score>
     <scoreDef xml:id="scoredef-000000176844426">
      <staffGrp xml:id="m-000000184953556">
       <staffDef xml:id="staffdef-000000031840020" clef.shape="G" clef.line="2" meter.count="4" meter.unit="4" n="1" lines="5" />
      </staffGrp>
     </scoreDef>
     <section xml:id="section-000000089094457">
      <measure xml:id="measure-L5" n="1">
       <staff xml:id="staff-L5F1" n="1">
        <layer xml:id="layer-L5F1" n="1">
         <beam xml:id="beam-000000091151767">
          <tuplet xml:id="tuplet-000000187676009" num="3" numbase="2" bracket.visible="false" num.format="count">
           <note xml:id="note-L6F1" dur="8" oct="4" pname="c" accid.ges="n" />
           <note xml:id="note-L7F1" dur="8" oct="4" pname="d" accid.ges="n" />
           <note xml:id="note-L8F1" dur="8" oct="4" pname="e" accid.ges="n" />
          </tuplet>
         </beam>
         <beam xml:id="beam-000000034448905">
          <tuplet xml:id="tuplet-000000021163507" num="3" numbase="2" bracket.visible="false" num.format="count">
           <rest xml:id="rest-L9F1" dur="8" />
           <rest xml:id="rest-L10F1" dur="8" />
           <rest xml:id="rest-L11F1" dur="8" />
          </tuplet>
         </beam>
         <beam xml:id="beam-000000032443929">
          <tuplet xml:id="tuplet-000000039026813" num="3" numbase="2" bracket.visible="false" num.format="count">
           <note xml:id="note-L12F1" dur="8" oct="4" pname="c" accid.ges="n" />
           <rest xml:id="rest-L13F1" dur="8" />
           <note xml:id="note-L14F1" dur="8" oct="4" pname="d" accid.ges="n" />
          </tuplet>
         </beam>
         <beam xml:id="beam-000000094031661">
          <tuplet xml:id="tuplet-000000056912280" num="3" numbase="2" bracket.visible="false" num.format="count">
           <rest xml:id="rest-L15F1" dur="8" />
           <note xml:id="note-L16F1" dur="8" oct="4" pname="e" accid.ges="n" />
           <rest xml:id="rest-L17F1" dur="8" />
          </tuplet>
         </beam>
        </layer>
       </staff>
      </measure>
      <measure xml:id="measure-L18" right="end">
       <staff xml:id="staff-L18F1" n="1">
        <layer xml:id="layer-L18F1" n="1">
         <beam xml:id="beam-000000146212304">
          <tuplet xml:id="tuplet-000000024657746" num="3" numbase="2" bracket.visible="false" num.format="count">
           <rest xml:id="rest-L19F1" dur="8" />
           <note xml:id="note-L20F1" dur="8" oct="4" pname="d" accid.ges="n" />
           <note xml:id="note-L21F1" dur="8" oct="4" pname="e" accid.ges="n" />
          </tuplet>
         </beam>
         <beam xml:id="beam-000000066625643">
          <tuplet xml:id="tuplet-000000079223481" num="3" numbase="2" bracket.visible="false" num.format="count">
           <note xml:id="note-L22F1" dur="8" oct="4" pname="c" accid.ges="n" />
           <note xml:id="note-L23F1" dur="8" oct="4" pname="d" accid.ges="n" />
           <rest xml:id="rest-L24F1" dur="8" />
          </tuplet>
         </beam>
         <beam xml:id="beam-000000088927177">
          <tuplet xml:id="tuplet-000000165203975" num="3" numbase="2" bracket.visible="false" num.format="count">
           <rest xml:id="rest-L25F1" dur="8" />
           <rest xml:id="rest-L26F1" dur="8" />
           <note xml:id="note-L27F1" dur="8" oct="4" pname="e" accid.ges="n" />
          </tuplet>
         </beam>
         <beam xml:id="beam-000000087645973">
          <tuplet xml:id="tuplet-000000106836414" num="3" numbase="2" bracket.visible="false" num.format="count">
           <note xml:id="note-L28F1" dur="8" oct="4" pname="c" accid.ges="n" />
           <rest xml:id="rest-L29F1" dur="8" />
           <rest xml:id="rest-L30F1" dur="8" />
          </tuplet>
         </beam>
        </layer>
       </staff>
      </measure>
     </section>
    </score>
   </mdiv>
  </body>
 </music>
</mei>
```

original Humdrum data

```
!!!filter: autobeam
**kern
*clefG2
*M4/4
=1-
12c
12d
12e
12r
12r
12r
12c
12r
12d
12r
12e
12r
=
12r
12d
12e
12c
12d
12r
12r
12r
12e
12c
12r
12r
==
*-
```

**Comment by rettinghaus** [comment]
Can this be closed now?

**Comment by rettinghaus** [comment]
fixed with #601

**Comment by lpugin** [comment]
Thanks
